tests/responses/aps/aps_single_response.json

```diff
@@ -64,6 +64,7 @@
             {
                "surname":"Alemi",
                "affiliationIds":[
+                  "a1",
```

Comment:
So, at the end we are not testing this anywhere, right? So no need to change it

Reply:
we are testing it, we see that the author does not have repetitive affiliations

Reply:
but where do we assert it?